includes.mk

```diff
@@ -0,0 +1,12 @@
+COMPONENT ?= workflow
```

Comment:
Perhaps. DEIS_REGISTRY, IMAGE and IMAGE_PREFIX are also only used by workflow so they're not shared as well. `make info` makes sense to keep in, but at the same time the client doesn't deploy an image or uses a dev reigstry so `make -C client info` would be displaying "useless" information as well.

I'm all for this PR if workflow and the client shared more variables together, but it seems like VERSION is the only candidate at this time, and even then they differ (for some unknown reason). Maybe we should revisit this once we figure out why the client version differs from the components? :)